Demo/src/main/java/com/braintreepayments/demo/SEPADebitFragment.java

```diff
@@ -46,15 +49,23 @@ public void launchSEPADebit(View view) {
 
         SEPADebitRequest request = new SEPADebitRequest();
         request.setAccountHolderName("John Doe");
-        request.setCustomerId("a-customer-id");
-        request.setIban("FR7618106000321234566666610");
+        request.setCustomerId(generateRandomCustomerId());
+        request.setIban(generateRandomIban());
```

Comment:
A specific IBAN/customer ID mandate can only be approved once, so randomly generating for continued testing.